eclipse-theia/theia
# Removed background of clear button

#### What it does

The svg for the clear button had a fixed background color. So it didn't look good on backgrounds with slightly different colors.
Before:
![Bildschirmfoto 2020-01-15 um 11 23 19](https://user-images.githubusercontent.com/28291421/72426376-e168a400-3789-11ea-9af8-3090d6aeba2f.png)
After:
![Bildschirmfoto 2020-01-15 um 11 23 55](https://user-images.githubusercontent.com/28291421/72426378-e168a400-3789-11ea-9671-c50c3e1f36d2.png)
  

#### How to test

Open theia. 
Change to Light theme. 
Open "Search In Workspace" Widget. 
Search for something so the "Clear" Button is enabled. 
The button should have an transparent background.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
